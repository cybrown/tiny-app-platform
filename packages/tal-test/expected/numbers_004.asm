main()
  entry:
    Literal           0
    Pop               inBlock: false
    Literal           1
    Pop               inBlock: false
    Literal           1
    Pop               inBlock: false
    Literal           202
    Pop               inBlock: false
    Literal           16
    Pop               inBlock: false
    Literal           10
    Pop               inBlock: false
    Literal           15
    Pop               inBlock: false
    Literal           15
    Pop               inBlock: false
    Local             G
    Pop               inBlock: false
    Literal           11205603
