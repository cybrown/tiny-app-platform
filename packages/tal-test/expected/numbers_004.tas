0b0
0b1
0b001
0b1100_1010
0x10
0xA
0xF
0xF
G
0xAA_FB_E3
