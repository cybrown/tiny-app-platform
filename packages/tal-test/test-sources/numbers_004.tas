0b0
0b1
0b001
0b1100_1010
0x10
0xA
0xF
0xFG
0xAA_FB_E3
