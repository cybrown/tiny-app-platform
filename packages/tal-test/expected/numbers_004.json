[
  0,
  1,
  1,
  202,
  16,
  10,
  15,
  15,
  {
    "kind": "Local",
    "name": "G"
  },
  11205603
]